api/lanes.py

```diff
@@ -699,6 +698,8 @@ def create_lane_for_small_collection(_db, library, parent, languages, priority=0
         genres=[],
     )
     language_identifier = LanguageCodes.name_for_languageset(languages)
+    if not language_identifier:
+        return 0
```

Comment:
In addition to my comment on the core branch, I'd suggest logging an error if this happens--otherwise it could be mysterious behavior.